test/unit/spec/common/cellComponents/fieldTableCellWidget-Spec.js

```diff
@@ -165,5 +165,172 @@ define([
             expect(result).toBeNull();
             this.fieldCellWidgetInstance = null;
         });
+
+        [
```

Comment:
A bunch of new test cases for the new functionality.
1. test that the various duplicate labels show up properly.
2. test that duplicate labels can go away when reset.
3. test that non-duplicate error validations work right.
4. test that those errors go away properly.
5. test toggling through all the different combos of input errors, duplicates, and dismissing each.